pkg/virt-handler/cache/cache.go

```diff
@@ -60,11 +60,8 @@ func (icp *iterableCheckpointManager) ListKeys() []string {
 		return []string{}
 	}
 
-	keys := []string{}
+	keys := make([]string, len(entries))
```

Comment:
this is not correct. the entry in the for below will be added after `len(entries)` empty strings
https://go.dev/play/p/t6rXDVBi8dH

Reply:
You are absolutely right! Great catch and :-1:  for me, I did this mistake once already